sumpy/test/test_misc.py

```diff
@@ -592,9 +592,9 @@ def get_pde_as_diff_op(self):
 def test_get_storage_index(order, knl, compressed):
     dim = knl.dim
     if compressed:
-        wrangler = LinearPDEBasedExpansionTermsWrangler(order, dim, knl=knl)
+        wrangler = LinearPDEBasedExpansionTermsWrangler(order, dim, None, knl=knl)
     else:
-        wrangler = FullExpansionTermsWrangler(order, dim)
+        wrangler = FullExpansionTermsWrangler(order, dim, None)
```

Comment:
```suggestion
        wrangler = FullExpansionTermsWrangler(order, dim, max_mi=None)
```
? Reads a bit better easier for those of us who are not as familiar with the interface :grin:

EDIT: Hm, this is in a test, so maybe nevermind..